packages/wallets/bitkeep/src/adapter.ts

```diff
@@ -37,7 +37,7 @@ export class BitKeepWalletAdapter extends BaseSignerWalletAdapter {
k2NSAzNjIuMTIgNTExLjk5IDM1NS43NTUgNTExLjk5IDM1MS4wNjVMNTEyIDE2MC45MzVaIiBmaWxsPSIjNzUyNEY5Ii8+CjxwYXRoIGZpbGwtcnVsZT0iZXZlbm9kZCIgY2xpcC1ydWxlPSJldmVub2RkIiBkPSJNNDE1LjEyNSAxNzcuMTU1VjE5NS4yMzVDNDE1LjEyNSAxOTYuOTg1IDQxNC42NjMgMTk4LjcwNCA0MTMuNzg1IDIwMC4yMThDNDEyLjkwNyAyMDEuNzMxIDQxMS42NDQgMjAyLjk4NiA0MTAuMTI1IDIwMy44NTVMMzUxLjg3IDIzNy41TDQwMy45MiAyNjcuNDY1QzQwNy4zMyAyNjkuNDI2IDQxMC4xNjIgMjcyLjI1MiA0MTIuMTMxIDI3NS42NTdDNDE0LjEgMjc5LjA2MiA0MTUuMTM2IDI4Mi45MjYgNDE1LjEzNSAyODYuODZWMzM1LjA2QzQxNS4xMzQgMzM4Ljk5OCA0MTQuMDk1IDM0Mi44NjYgNDEyLjEyMSAzNDYuMjc0QzQxMC4xNDggMzQ5LjY4MiA0MDcuMzEgMzUyLjUwOSA0MDMuODk1IDM1NC40N0wyNjcgNDMzLjAzQzI2My42MDEgNDM0Ljk4MSAyNTkuNzUgNDM2LjAwNSAyNTUuODMxIDQzNkMyNTEuOTEyIDQzNS45OTQgMjQ4LjA2MyA0MzQuOTYgMjQ0LjY3IDQzM0wxOTkuODUgNDA3LjEyQzE5OS4wOTMgNDA2LjY4NCAxOTguNDY0IDQwNi4wNTYgMTk4LjAyNyA0MDUuM0MxOTcuNTg5IDQwNC41NDQgMTk3LjM1OSA0MDMuNjg2IDE5Ny4zNTkgNDAyLjgxMkMxOTcuMzU5IDQwMS45MzkgMTk3LjU4OSA0MDEuMDgxIDE5OC4wMjcgNDAwLjMyNEMxOTguNDY0IDM5OS41NjggMTk5LjA5MyAzOTguOTQxIDE5OS44NSAzOTguNTA1TDM0Ny43ODUgMzEzLjM1NUMzNDguMTYgMzEzLjEzNCAzNDguNDcyIDMxMi44MTkgMzQ4LjY4OCAzMTIuNDQxQzM0OC45MDQgMzEyLjA2MyAzNDkuMDE4IDMxMS42MzUgMzQ5LjAxOCAzMTEuMkMzNDkuMDE4IDMxMC43NjQgMzQ4LjkwNCAzMTAuMzM3IDM0OC42ODggMzA5Ljk1OUMzNDguNDcyIDMwOS41ODEgMzQ4LjE2IDMwOS4yNjYgMzQ3Ljc4NSAzMDkuMDQ1TDI5Mi44MzUgMjc3LjMxNUMyOTEuMzI0IDI3Ni40NDMgMjg5LjYxIDI3NS45ODQgMjg3Ljg2NSAyNzUuOTg0QzI4Ni4xMiAyNzUuOTg0IDI4NC40MDYgMjc2LjQ0MyAyODIuODk1IDI3Ny4zMTVMMTMyLjU4IDM2My45MDVDMTMxLjQ1IDM2NC41NTUgMTMwLjE2OSAzNjQuODk2IDEyOC44NjUgMzY0Ljg5NkMxMjcuNTYxIDM2NC44OTYgMTI2LjI4IDM2NC41NTUgMTI1LjE1IDM2My45MDVMMTA4LjE1IDM1NC4xNjVDMTA0LjcyNyAzNTIuMjA1IDEwMS44ODIgMzQ5LjM3NSA5OS45MDM4IDM0NS45NjJDOTcuOTI1NiAzNDIuNTQ5IDk2Ljg4NDIgMzM4LjY3NSA5Ni44ODUgMzM0LjczVjMxNUM5Ni44ODcxIDMxMy42ODkgOTcuMjM0MyAzMTIuNDAyIDk3Ljg5MTggMzExLjI2OEM5OC41NDkyIDMxMC4xMzQgOTkuNDkzNiAzMDkuMTkzIDEwMC42MyAzMDguNTRMMzE2LjA1NSAxODQuNzNDMzE2LjQyOSAxODQuNTA5IDMxNi43MzkgMTg0LjE5NCAzMTYuOTU1IDE4My44MTdDMzE3LjE3IDE4My40MzkgMzE3LjI4NCAxODMuMDEyIDMxNy4yODQgMTgyLjU3N0MzMTcuMjg0IDE4Mi4xNDMgMzE3LjE3IDE4MS43MTYgMzE2Ljk1NSAxODEuMzM4QzMxNi43MzkgMTgwLjk2MSAzMTYuNDI5IDE4MC42NDYgMzE2LjA1NSAxODAuNDI1TDI2MS4wNTUgMTQ4LjU4NUMyNTkuNTQzIDE0Ny43MTQgMjU3LjgzIDE0Ny4yNTYgMjU2LjA4NSAxNDcuMjU2QzI1NC4zNCAxNDcuMjU2IDI1Mi42MjcgMTQ3LjcxNCAyNTEuMTE1IDE0OC41ODVMMTA0LjMyIDIzM0MxMDMuNTYyIDIzMy40MjcgMTAyLjcwNiAyMzMuNjQ5IDEwMS44MzYgMjMzLjY0NEMxMDAuOTY2IDIzMy42MzkgMTAwLjExMyAyMzMuNDA3IDk5LjM2MDEgMjMyLjk3MkM5OC42MDczIDIzMi41MzYgOTcuOTgxMSAyMzEuOTExIDk3LjU0MzUgMjMxLjE1OUM5Ny4xMDYgMjMwLjQwOCA5Ni44NzIxIDIyOS41NTUgOTYuODY1IDIyOC42ODVWMTc2LjkyNUM5Ni44NjI0IDE3Mi45ODQgOTcuOTAwNSAxNjkuMTEzIDk5Ljg3NDEgMTY1LjcwMkMxMDEuODQ4IDE2Mi4yOTEgMTA0LjY4NyAxNTkuNDYyIDEwOC4xMDUgMTU3LjVMMjQ1IDc4Ljk3QzI0OC4zOTYgNzcuMDE5MiAyNTIuMjQ1IDc1Ljk5NDYgMjU2LjE2MSA3NS45OTlDMjYwLjA3NyA3Ni4wMDM0IDI2My45MjQgNzcuMDM2NiAyNjcuMzE1IDc4Ljk5NUw0MDMuOTI1IDE1Ny43OEM0MDcuMzI4IDE1OS43NDMgNDEwLjE1NCAxNjIuNTY2IDQxMi4xMiAxNjUuOTY3QzQxNC4wODYgMTY5LjM2OCA0MTUuMTIyIDE3My4yMjcgNDE1LjEyNSAxNzcuMTU1WiIgZmlsbD0id2hpdGUiLz4KPC9zdmc+Cg==';
```

Comment:
Based on the size of the encoded data, this looks like it may be a PNG encoded inside an SVG. Is it?

Reply:
No, it's base46 directly converted by SVG